strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generation/PackageJson.java

```diff
@@ -0,0 +1,163 @@
+package org.carlspring.strongbox.artifact.generation;
+
+import java.util.ArrayList;
+
+import org.json.JSONArray;
+
+public class PackageJson
```

Comment:
@despoinasr : It would probably be better to add this class to strongbox/strongbox-npm-metadata .
@sbespalov : Any thoughts on this?